content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,194 @@
+---
+title: April '21 Community Gems
+date: 2021-04-30
+description: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running cml pipeline reports and more.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. 
+  This month: remote storage integration, removing old experiments,
+  ideas for running cml pipeline reports  and more.
+picture: 2021-04-30/gems-cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/april-21-community-gems/
+tags:
+  - Discord
+  - Gems
+  - DVC
+  - CML
+  - Remote Storage
+---
+
+Each month we go through our Discord messages to pull out some of the best
+questions from our community. AKA: Community Gems. 💎 This month we'd like to
+thank asraniel, Julian, mattlbeck, Ahti, **\_**, yikeqicn, lexzen, EdAb,
+FreshLettuce for inspiring this month's gems!
+
+As always, [join us in Discord](https://discord.com/invite/dvwXA2N) to get all
+your DVC and CML questions answered!
+
+## DVC
+
+### [Does the directory system have to already be set up before pushing to OwnCloud with a WebDav remote?](https://discord.com/channels/485586884165107732/485596304961962003/831472645508694046)
```

Comment:
I'm not sure I understand what `directory system` is? 

In this specific case, probably better to rephrase the question into something like

"I'm getting .. Resource not found ... when I try to use OwnCloud via webdav. What would be the reason for this .."

put the link to the actual error log

Reply:
I'm not sure if we have a link to the actual error, but I've tried to clear up this text. @jendefig Do you know if we have a link to the error log?